BSP/Inc/BSP_CAN.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * Header file for the library to interact
+ * with both CAN lines in the car
+ */
+
+#ifndef __BSP_CAN_H
+#define __BSP_CAN_H
+
+#include "common.h"
+#include <bsp.h>
```

Comment:
For consistency, it might be better to use `#include "bsp.h"` to distinguish our code from system headers.

Reply:
I'll go back through and change that in all the parts of the simulator once everything is merged in